src/components/Input/Input.styles.ts

```diff
@@ -80,7 +142,9 @@ const StyledInput = styled.input<{
 
   &:disabled{
     background-color:${theme.core.background.bgSecondary};
+    color${theme.core.content.contentTertiary}
```

Comment:
Question; is there meant to be a colon between `color` and the colour value ?

Reply:
Is there meant to be a semi-colon at end of expression?

Reply:
yup fixed it! :) thanks!